app/src/main/java/com/fer1592/k8s_android_console/view/activities/MainActivity.kt

```diff
@@ -12,7 +12,7 @@ import com.fer1592.k8s_android_console.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     private var _binding: ActivityMainBinding? = null
-    private val binding get() = _binding!!
+    val binding get() = _binding!!
```

Comment:
should never set binding as public, it must be handled only by the main activity. We can write some public methods that modify the binding state

Reply:
updated and fixed in theory!